v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/EntityHelperImpl.java

```diff
@@ -806,6 +806,27 @@ public int mapInternalEntityDataName(Entity entity, String name) {
         return EntityDataNameMapper.getIdForName(((CraftEntity) entity).getHandle().getClass(), name);
     }
 
+    public static final Field SynchedEntityData_itemsById = ReflectionHelper.getFields(SynchedEntityData.class).get(ReflectionMappingsInfo.SynchedEntityData_itemsById);
+
+    public static Int2ObjectMap<SynchedEntityData.DataItem<Object>> getDataItems(Entity entity) {

```

Comment:
should update `modifyInternalEntityData` to use this

Reply:
Ye, haven't touched the existing `internal_data` mechanism yet as I haven't finished finalizing how the new methods are setup